sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -858,11 +857,11 @@ export class MetricsAdvisorAdministrationClient {
     createDataSourceCredential(dataSourceCredential: DataSourceCredentialEntityUnion, options?: OperationOptions): Promise<DataSourceCredentialEntityUnion>;
     createDetectionConfig(config: Omit<AnomalyDetectionConfiguration, "id">, options?: OperationOptions): Promise<AnomalyDetectionConfiguration>;
     createHook(hookInfo: EmailNotificationHook | WebNotificationHook, options?: OperationOptions): Promise<NotificationHookUnion>;
-    deleteAlertConfig(id: string, options?: OperationOptions): Promise<RestResponse>;
-    deleteDataFeed(id: string, options?: OperationOptions): Promise<RestResponse>;
-    deleteDataSourceCredential(id: string, options?: OperationOptions): Promise<RestResponse>;
-    deleteDetectionConfig(id: string, options?: OperationOptions): Promise<RestResponse>;
-    deleteHook(id: string, options?: OperationOptions): Promise<RestResponse>;
+    deleteAlertConfig(id: string, options?: OperationOptions): Promise<void>;
+    deleteDataFeed(id: string, options?: OperationOptions): Promise<void>;
+    deleteDataSourceCredential(id: string, options?: OperationOptions): Promise<void>;
+    deleteDetectionConfig(id: string, options?: OperationOptions): Promise<void>;
+    deleteHook(id: string, options?: OperationOptions): Promise<void>;
```

Comment:
This is a breaking change.
I think we had some discussion around this in https://github.com/Azure/azure-sdk-for-js/issues/15916

Can we have something in the convenience layer that calls the callback to get the rawresponse and form the expected `RestResponse` here?  Once you figure that out, please update this wiki as well: https://github.com/Azure/autorest.typescript/wiki/%60core-http%60-dependency-migration-to-%60core-client%60-%60core-rest-pipeline%60#change-to-the-_response-property

Reply:
Do you mean something like this? - https://github.com/KarishmaGhiya/azure-sdk-for-js/blob/061efa38b29d10860d66561bd2f516a969572293/sdk/monitor/monitor-query/src/internal/modelConverters.ts#L80

Reply:
https://github.com/KarishmaGhiya/azure-sdk-for-js/blob/061efa38b29d10860d66561bd2f516a969572293/sdk/monitor/monitor-query/src/logsQueryClient.ts#L139

Reply:
I think so...